Library/Formula/git.rb

```diff
@@ -1,7 +1,7 @@
 class Git < Formula
   homepage "http://git-scm.com"
```

Comment:
Cool, I'll keep that in mind for the future. I also just reread contributing.md and as a result squashed that change into one commit. let me know if there's anything else you spot.